README.md

```diff
@@ -1,3 +1,190 @@
+- [Contributing](#contributing)
+
+## Introduction
+The CI/CD Scan by AccuKnox is a powerful tool designed to enhance the security of 
+your continuous integration and deployment pipelines. By providing comprehensive 
+assessment, monitoring, and protection capabilities, it enables you to:
+
+- Detect and prevent potential supply chain attacks in CI/CD environments
+- Gain real-time visibility into your pipeline's security posture
+- Streamline the integration of security practices into your DevOps workflow
+
+In today's fast-paced software development landscape, where operational efficiency 
+is paramount, the CI/CD Scan by AccuKnox empowers DevSecOps teams to deploy with confidence. 
+
+## KubeArmor
+KubeArmor is a CNCF project that powers this action to scan the events taking place
+in the CI/CD environment such as GitHub runner. KubeArmor uses eBPF monitors to 
+watch over the events taking place at system level and uses LSMs (Linux security modules)
+for enforcing the security policies either in block or audit mode. 
+
+To learn more about KubeArmor please visit, https://kubearmor.io/ 
+
+We install KubeArmor in systemd mode in the GitHub runner and that lets us watch over
+the events and enforce security policies safely. 
+
+## Features
+This GitHub action provides you with the necessary information needed for you to 
+monitor and view as the events take place in the CI/CD environment. 
+
+### 1) Policy driven alert generation
+We apply standard industry policies as baseline security posture of the CI/CD 
+environment. This helps you to quickly identify if there is a process, network call
+or file event that is taking place that should be audited or blocked. 
+
+We support and apply multiple compliance based security policies that includes 
+![MITRE ATT&CK](https://avatars.githubusercontent.com/u/43272913?s=200&v=4)|![NIST](https://avatars.githubusercontent.com/u/5330158?s=200&v=4)|![CIS](https://avatars.githubusercontent.com/u/8919353?s=200&v=4)
+---------------------------------------------------------------------------|------------------------------------------------------------------|----------------------------------------------------------------
+MITRE ATT&CK | NIST | CIS
+
+All of our security policy templates are open sourced and can be found here: [Policy Templates](https://github.com/kubearmor/policy-templates)
+
+The alerts generated after the scan of your CI/CD pipeline is completed looks like this
+![](./assets/alert_example_1.png)
+
+It clearly tells you about the policy itself, the command which caused the generation of alert
+and the message itself, which makes it easy for you to understand the nature of alert and
+a possible issue in the CI/CD pipeline itself.
+
+### 2) Define your own security policies
+Other than providing the baseline as an initial security posture, you can also write your
+own security policies and have a more granular control on the CI/CD pipeline. 
+
+Here is an example of a security policy
```

Comment:
Remove the below YAML policy and provide a realistic policy from existing template

Reply:
Added a more realistic policy.